### PR TITLE
util/pingpong: added tagged pingpong

### PR DESCRIPTION
**Pull request by hoopoepg**
- added send/recv using tagged interface

Signed-off-by: Oblomov, Sergey <sergey.oblomov@intel.com>

### CODE REVIEW

File: util/pingpong.c [hunk 1]
```diff
@@ -2161,6 +2187,8 @@ int main(int argc, char **argv)
 		}
 	}
 
+	ct.hints->caps = FI_MSG | (ct.tagged ? FI_TAGGED : 0);
```
Comment (shefty) [inline util/pingpong.c]: Is FI_MSG needed if tagged is selected?

File: util/pingpong.c [hunk 2]
```diff
@@ -1953,6 +1977,9 @@ void pp_parse_opts(struct ct_pingpong *ct, int op, char *optarg)
 		ct->opts.dst_port = parse_ulong(optarg, UINT16_MAX);
 		break;
 
+	case 'm':
+		ct->tagged = strncasecmp("msg", optarg, 4);
```
Comment (shefty) [inline util/pingpong.c]: Can we drop ct->tagged and set ct.hints.caps = FI_TAGGED directly?

File: util/pingpong.c (hunk 1 above)
Comment (hoopoepg) [inline util/pingpong.c]: MSG is used in finalize

Comment (shefty) [inline util/pingpong.c]: thanks

File: util/pingpong.c (hunk 2 above)
Comment (hoopoepg) [inline util/pingpong.c]: yes, but it is simpler to maintain when initialization of hints is located at single function

Comment (shefty) [inline util/pingpong.c]: The call this is in already sets various hints, like domain name, changes endpoint type, etc.  I think we're okay modifying the caps as well.  I'd rather have a single variable to check than maintaining two that mirror each other.

Comment (hoopoepg) [inline util/pingpong.c]: ok, updated

**Pull request merged**